packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -839,7 +839,9 @@ abstract class NetworkImage extends ImageProvider<NetworkImage> {
 ///
 /// See also:
 ///
-///  * [Image.file] for a shorthand of an [Image] widget backed by [FileImage].
+///  * [Image.file] for a shorthand of an [Image] widget backed by [FileImage],
+///     which creates an in memory copy of file retained in the [ImageCache].
```

Comment:
I think it would make sense to just explain in the docs above this they the provider does not monitor the file for changes and to call the [evict] method if the underlying image data changes.

Reply:
Updated, thanks :)